.github/workflows/publish.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow is triggered two ways:
+#
+# 1. When a tag is created, the workflow will upload the package to
+#    test.pypi.org.
+# 2. When a release is made, the workflow will upload the package to pypi.org.
+#
+# It is done this way until PyPI has draft reviews, to allow for a two-stage
+# upload with a chance for manual intervention before the final publication.
+name: Upload package
+
+on:
+  release:
+    types: [created]
+  push:
+    tags:
+      - '*'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: "__token__"
```

Comment:
@pganssle FWIW I'd put this specific var on the job level to reduce copy-paste: https://github.com/python/tzdata/pull/18/files#r420771038

Reply:
Where is the copy-paste coming in? It only gets set in the "Publish package" environment, no?

Reply:
Copy-paste appears with the first suggestion that I sent